components/form/form-element-mixin.js

```diff
@@ -77,21 +77,34 @@ export class FormElementValidityState {
 export const FormElementMixin = superclass => class extends LocalizeCoreElement(superclass) {
 
 	static get properties() {
-		return { validationError: { attribute: false, type: String } };
+		return {
+			forceInvalid: { type: Boolean, attribute: false },
+			invalid: { type: Boolean, reflect: true },
```

Comment:
Does `invalid` need to reflect or be bound to an attribute at all? Seems like it's almost more of a computed property -- its value is based on whether there's a validation error or `forceInvalid` is set. So we don't want consumers seeing it being sprouted as an attribute on elements or thinking they should be setting it themselves.

Reply:
The idea was to have some attribute that could be used specifically for styling components when they're in an invalid state. Similar to `:invalid` except with an attribute selector because we can't create our own pseudo-class selector. Forcing people to reflect it themselves becomes a bit awkward.

Reply:
Ah ok gotcha, that makes sense! `attribute: false` _might_ still make sense then... reflect will cause it to sprout an attribute for CSS selectors, but then `attribute: false` I think will mean that if they manually change that attribute it won't have any effect on the property, which I think is what you want.

Reply:
Unfortunately `attribute: false` seems to have issues with `reflect`, this is from the [docs](https://lit-element.polymer-project.org/guide/properties#conversion):
>  If attribute is false, the converter, reflect and type options are ignored.

Reply:
Boo. Ok thanks for investigating!